src/block_ffm.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code,unused_imports, unused_mut, invalid_value)]
```

Comment:
Why does this need to be in each file? shouldn't it only be in the lib & main?

Reply:
Tried that, didn't work as expected unfortunately